server/actions/email.js

```diff
@@ -2,9 +2,9 @@ import fetch from "isomorphic-unfetch";
 
 const sendGridUrl = "https://api.sendgrid.com/v3/mail/send";
 
-export async function sendContactEmail(email, name, message) {
-  if (email == null || name == null || message == null) {
-    throw new Error("Email, name, and message must be provided");
+export async function sendEmail(recipient = process.env.TO_ADDRESS, subject, body, mimetype = "text/plain") {
```

Comment:
These defaulted parameters cause more confusion than they're worth having. Better to have them be explicitly set each time.